lib/puppet/ssl/host.rb

```diff
@@ -419,6 +430,138 @@ def load_crls(path)
     end
   end
 
+  # Ensures that the CA certificate is available for either generating or
+  # validating the host's cert.
+  # It will first check if the cert is present in memory (used for testing),
+  # then check on disk, and finally try to download it.
+  # @return true if the CA certificate was found, false otherwise
+  def ensure_ca_certificate
+    file_path = certificate_location(CA_NAME)
+    if check_for_certificate_in_memory(CA_NAME)
+      true
+    elsif Puppet::FileSystem.exist?(file_path)
+      load_certificate_bundle(Puppet::FileSystem.read(file_path))
```

Comment:
Is this load necessary? `load_certificate_bundle` returns an array of pems, but, since we aren't doing anything with that array here, maybe we don't need it?

Reply:
Updated with a comment.